os/board/common/partitions.c

```diff
@@ -36,11 +37,12 @@
 #endif
 #include "common.h"
 
+#define FS_PATH_MAX 15
+
 #ifdef CONFIG_FLASH_PARTITION
 struct partition_data_s g_flash_part_data = {
 	CONFIG_FLASH_PART_TYPE,
 	CONFIG_FLASH_PART_SIZE,
-	CONFIG_FLASH_MINOR,
```

Comment:
1. Please remove CONFIG_FLASH_MINOR from all of defconfigs.
2. Please remove config FLASH_MINOR from os/board/common/Kconfig.

Reply:
Done. Thanks.